### PR TITLE
Add simple converter functions for converting nominal to real terms

### LINKED ISSUE #36: Costs converter to real terms is needed
Since we use nominal values for costs as an input, we need to have some kind of converter to convert it back to real terms.
To be more precise, two scripts are needed:
1. Converting power prices (or other hourly time series) for a given year to real terms.
2. Converting costs input data given in annual resolution to real terms.

### PR DESCRIPTION
**Pull request by jokochems**
Add helpers to allow converting costs / prices given in nominal terms to real terms.

### CODE REVIEW

**Comment by jokochems** [comment]
@hannesfelipe: This is a very simple one. It would be nice if you could review it. Thank you! 👍

File: pommesdispatch/model_funcs/helpers.py [hunk 1]
```diff
@@ -120,7 +118,80 @@ def convert_annual_limit(annual_limit, start_time, end_time):
...
+        if column != "label":
+            try:
+                real_costs[column] = real_costs[column].div(
+                    inflation_rate ** (int(column) - year)
+                )
+            except TypeError:
+                msg = (
+                    "DataFrame format not as expected\n"
+                    "Except for column 'label', "
+                    "all other columns must be integer"
```
Comment (hannesfelipe) [inline pommesdispatch/model_funcs/helpers.py]: Might be helpful to specify that the column name must be an integer, as it is the years?
Values of the rows can be floats as tehy are tho cost. Thus, no confusion is possible.

File: pommesdispatch/model_funcs/helpers.py [hunk 2]
```diff
@@ -120,7 +118,80 @@ def convert_annual_limit(annual_limit, start_time, end_time):
...
+    real_time_series: :obj:`pd.DataFrame`
+        Real time series data in hourly resolution
+    """
+    if not type(nominal_time_series.index.year == pd.DatetimeIndex):
+        raise TypeError("Given time series must have a pd.DatetimeIndex!")
+
+    time_series_year = nominal_time_series.index.year[0]
+    real_time_series = nominal_time_series.div(
+        inflation_rate ** (time_series_year - year)
+    )
```
Comment (hannesfelipe) [inline pommesdispatch/model_funcs/helpers.py:195]: Does it also make sense to include the check for integer values for the years?

**Review by hannesfelipe — CHANGES_REQUESTED** [review]
Formulas look good. Two more comments on the error messages.

File: pommesdispatch/model_funcs/helpers.py (hunk 1 above)
Comment (jokochems) [reply pommesdispatch/model_funcs/helpers.py]: Yes, you are right. The column name must be an integer. The dtype is probably float. I'll adjust the error message. Thank you!

File: pommesdispatch/model_funcs/helpers.py (hunk 2 above)
Comment (jokochems) [reply pommesdispatch/model_funcs/helpers.py:195]: Hmm, in my view, this should be ensured by the user of the function and is stated in the docstring. 
Also, you can easily notice that from the error being thrown when you put in a non-numeric data type.

In principle, it is also possible to include explicit type checks in python. It is not so common, though, I think.

So something like
```python3

def my_function(integer_attr: int) -> float:
    """Function does fancy stuff with an integer input and returns a float"""
    return integer_attr + 0.2
```
is a thing you can do.

**Comment by jokochems** [comment]
@hannesfelipe Would you please quickly re-review and press merge if everything is fine?

**Comment by jokochems** [comment]
> @hannesfelipe Would you please quickly re-review and press merge if everything is fine?

@hannesfelipe Would you please approve? Thank you!

**Review by hannesfelipe — APPROVED** [review]

**Pull request merged**